go.mod

```diff
@@ -1,9 +1,10 @@
 module github.com/kailt/imageresizer
 
 require (
-	github.com/aws/aws-sdk-go v1.15.59
-	github.com/cloudflare/tableflip v0.0.0-20181019105324-78281f93d075
+	github.com/aws/aws-sdk-go v1.15.64
+	github.com/cloudflare/tableflip v0.0.0-20181024123920-3728e38f7e51
```

Comment:
`aws-sdk-go` and `tableflip` are not part of the fsnotify feature and hence the 2 lines above should not be changed.